### DISCUSSION TITLE
Shared Matrix update throws telemetry:Batching LengthToBig error

### DISCUSSION THREAD

**Discussion by pashrafi1** [microsoft/FluidFramework#10148] [General]
In our application we have a shared matrix with a few hundreds rows. Whenever I try to update more than 128 rows in the matrix, I'm getting a telemetry error in console which reads fluid:telemetry:Batching LengthToBig{ "length":  *** , "threshold: 128 ...}. Is there anyway to change this threshold? 
(*** is any number greater than 128)

**Comment by tylerbutler** [comment]
@DLehenbauer @microsoft/fluid-cr-dds Can anyone answer this?

**Reply by pashrafi1** [reply]
The  batchLengthThreshold is set in [BatchTracker](https://github.com/microsoft/FluidFramework/blob/main/packages/runtime/container-runtime/src/batchTracker.ts). Is there any way that we can override this threshold? Or is there any way to send updates in 128 rows operations

![image](https://user-images.githubusercontent.com/74574725/168632756-fad58aee-8a62-41ad-84e9-953475ef9efd.png)
